assets/aml-benchmark/components/batch_benchmark_inference_with_inference_compute/spec.yaml

```diff
@@ -0,0 +1,281 @@
+jobs:
+  # Preparer
+  batch_inference_preparer: 
+    type: command
+    component: azureml:batch_inference_preparer@latest
+    inputs:
+      input_dataset: ${{parent.inputs.input_dataset}}
+      model_type: ${{parent.inputs.model_type}}
+      batch_input_pattern: ${{parent.inputs.batch_input_pattern}}
+      is_performance_test: ${{parent.inputs.is_performance_test}}
+      n_samples: ${{parent.inputs.n_samples}}
+      endpoint_url: ${{parent.inputs.endpoint_url}}
+      label_column_name: ${{parent.inputs.label_column_name}}
+      additional_columns: ${{parent.inputs.additional_columns}}
+    outputs:
+      formatted_data:
+        type: mltable
+      ground_truth_metadata:
+        type: uri_folder
+  # Config generation
+  config_generation:
+    type: command
+    component: azureml:batch_benchmark_config_generator@latest
+    inputs:
+      scoring_url: ${{parent.inputs.endpoint_url}}
+      connection_name: ${{parent.inputs.connections_name}}
+      deployment_name: ${{parent.inputs.deployment_name}}
+      authentication_type: ${{parent.inputs.authentication_type}}
+      debug_mode: ${{parent.inputs.debug_mode}}
+      additional_headers: ${{parent.inputs.additional_headers}}
+      ensure_ascii: ${{parent.inputs.ensure_ascii}}
+      max_retry_time_interval: ${{parent.inputs.max_retry_time_interval}}
+      initial_worker_count: ${{parent.inputs.initial_worker_count}}
+      max_worker_count: ${{parent.inputs.max_worker_count}}
+      configuration_file: ${{parent.inputs.endpoint_config_file}}
+      model_type: ${{parent.inputs.model_type}}
+      app_insights_connection_string: ${{parent.inputs.app_insights_connection_string}}
+    outputs:
+      batch_score_config:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.json
+  # Inference
+  endpoint_batch_score:
+    type: parallel
+    component: azureml:batch_score_llm@latest
+    inputs:
+      async_mode: False
+      data_input_table: ${{parent.jobs.batch_inference_preparer.outputs.formatted_data}}
+      configuration_file: ${{parent.jobs.config_generation.outputs.batch_score_config}}
+    outputs:
+      job_output_path:
+        type: uri_file
+      mini_batch_results_output_directory:
+        type: uri_folder
+    resources:
+      instance_count: ${{parent.inputs.instance_count}}
+    max_concurrency_per_instance:  ${{parent.inputs.max_concurrency_per_instance}}
+    mini_batch_size: 100KB #${{parent.inputs.mini_batch_size}}
+    retry_settings:
+      timeout: 6000
+      max_retries: 10
+    compute: ${{parent.inputs.inference_compute}}
+    environment_variables:
+      BATCH_SCORE_INITIAL_REQUEST_TIMEOUT: '4'
+      BATCH_SCORE_DELAY_AFTER_SUCCESSFUL_REQUEST: 'False'
+      BATCH_SCORE_MAX_REQUEST_TIMEOUT: '300'
+      
+  # Reformat
+  batch_output_formatter: 
+    type: command
+    component: azureml:batch_score_llm@latest
```

Comment:
same with component name.

Reply:
I have updated it. Thanks for catching this!